src/test/spec/crud.rs

```diff
@@ -18,6 +18,12 @@ fn test_predicate(test: &TestCase) -> bool {
     // The Rust driver doesn't support unacknowledged writes.
     let lower = test.description.to_lowercase();
 
-    // TODO: RUST-1071, RUST-1215: unskip comment tests
-    !lower.contains("unacknowledged") && !lower.contains("comment")
+    // TODO: RUST-1071, RUST-663: unskip comment tests
```

Comment:
Can we split these TODO comments to be above the relevant boolean checks? That'll make it a little easier for whoever does those tickets to know what to delete.